build-scripts/aur-git/50-systemd-user.conf

```diff
@@ -12,5 +12,5 @@ exec systemctl --user import-environment DISPLAY \
 exec hash dbus-update-activation-environment 2>/dev/null && \
      dbus-update-activation-environment --systemd DISPLAY \
                                                   SWAYSOCK \
-                                                  XDG_CURRENT_DESKTOP=sway \
+                                                  XDG_CURRENT_DESKTOP="swayfx:sway:wlroots" \
```

Comment:
Does having it colon separated actually work? Will it be detected by the portals conf?

Reply:
Portals work fine on my end. If I initiate screen sharing from Firefox/Discord, it calls xdp-wlr. Everything else calls xdp-gtk (as defined in `swayfx-portals.conf`).

The colon-separated list is used by `NotShowIn=`/`OnlyShowIn=` in `.desktop` files: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html

>  If `$XDG_CURRENT_DESKTOP` is set then it contains a colon-separated list of strings. In order, each string is considered. If a matching entry is found in `OnlyShowIn` then the desktop file is shown. If an entry is found in `NotShowIn` then the desktop file is not shown. If none of the strings match then the default action is taken (as above).
> `$XDG_CURRENT_DESKTOP` should have been set by the login manager, according to the value of the `DesktopNames` found in the session file. The entry in the session file has multiple values separated in the usual way: with a semicolon.